app/views/shared/views/ems_common/angular/_form.html.haml

```diff
@@ -106,7 +121,7 @@
                             "name"                    => "subscription",
                             "ng-model"                => "emsCommonModel.subscription",
                             "checkchange"             => "",
-                            "required"                => "",
+                            "required"                => true,
```

Comment:
@djberg96 I can't think of a case when Azure provider could be added without subscription ID, hence I took liberty to mark it required. Can you please confirm this will not break you anything for Azure cloud?